### PR TITLE
fixes #321 - move probe details title

### LINKED ISSUE #321: Add breadcrumb to 'data for this probe is unavailable' page
When looking at a probe's data, there is an 'explore/<probe_token>' breadcrumb indicator at the top of the main content area. This doesn't show up on the 'data unavailable' page that shows upwhen the probe has no data. This means there's no place on the page that shows the token id of the probe the user is trying to view.

### PR DESCRIPTION
**Pull request by spasovski**
Surprisingly not simple.

### CODE REVIEW

**Comment by openjck** [comment]
**IF** we care about eslint, which may be an open question (you made a good point that it's less important with compiled languages), I wanted to note that this brings our eslint problem count up from 228 to 230, with the following being new:

```
[...]/glam/src/components/regions/ProbeTitle.svelte
  1:9  error  Too many blank lines at the beginning of file. Max of 1 allowed  no-multiple-empty-lines

[...]/glam/src/routing/pages/probe/Explore.svelte
  8:74  error  Trailing spaces not allowed  no-trailing-spaces
```

File: src/components/errors/DataError.svelte [hunk 1]
```diff
@@ -7,9 +7,7 @@ export let moreInformation;
 
 <style>
 .data-error-msg {
-  /* background-color: var(--cool-gray-050); */
```
Comment (openjck) [inline src/components/errors/DataError.svelte:10]: 👍

File: src/components/errors/DataError.svelte [hunk 2]
```diff
@@ -18,13 +16,12 @@ export let moreInformation;
...
   --error-msg-width: 400px;
   --error-msg-color: var(--cool-gray-500);
-  margin-top: var(--space-8x);
+  margin-top: var(--space-16x);
   margin-bottom: var(--space-2x);
 }
 
 .data-error-msg__bg {
   background: radial-gradient(var(--cool-gray-100), var(--cool-gray-100));
-  /* box-shadow: inset 0px 0px 10px rgba(0,0,0,.1); */
```
Comment (openjck) [inline src/components/errors/DataError.svelte:27]: 👍

File: src/routing/pages/probe/Explore.svelte [hunk 3]
```diff
@@ -21,6 +22,7 @@
 
 <Probe let:data let:probeType>
   <div in:fade class="graphic-body__content">
+    <ProbeTitle view={$store.route.view} probeName={$store.probe.name} />
```
Comment (openjck) [inline src/routing/pages/probe/Explore.svelte:25]: Do we think there will be cases where `view` is equal to something other than `$store.route.view`? If not, would it make sense to consider not taking this as a prop at all, but instead accessing `$store.route.view` from the `ProbeTitle` component?

**Review by openjck — CHANGES_REQUESTED** [review]
Otherwise this is good. I like it!

File: src/components/explore/ProportionExplorerView.svelte [hunk 4]
```diff
@@ -91,8 +88,8 @@ h2 {
       {/if}
     </div>
 
-    <div class=body-control-set>
-      <label class=body-control-set--label>Categories</label>
+    <div class="body-control-set">
```
Comment (hamilton) [inline src/components/explore/ProportionExplorerView.svelte:91]: I'm seeing a lot of changes like this. Should we make this a lint rule so it is enforced? I have no preference but I typically don't put quotes in out of habit

**Pull request closed without merging**